README.md

```diff
@@ -1,226 +1,187 @@
-                o.onreadystatechange = function() {
-                    if (o.readyState === "loaded") {
-                        oTrackinginit();
-                    }
-                };
-            } else {
-                o.onload = oTrackinginit;
-            }
-            s.parentNode.insertBefore(o, s);
-        }
-    </script>
-</head>
-<body>
-<!-- Add fallback if browsers don't cut the mustard -->
-<div class="o-tracking o--if-no-js" data-o-component="o-tracking">
-    <div style="background:url('https://spoor-api.ft.com/px.gif?data=%7B%22category%22:%22page%22,%20%22action%22:%22view%22,%20%22system%22:%7B%22apiKey%22:%22qUb9maKfKbtpRsdp0p2J7uWxRPGJEP%22,%22source%22:%22o-tracking%22,%22version%22:%221.0.0%22%7D,%22context%22:%7B%22product%22:%22ft.com%22,%22content%22:%7B%22asset_type%22:%22page%22%7D%7D%7D');"></div>
-</div>
-<!-- Send an event -->
-<button onclick="document.body.dispatchEvent(new CustomEvent('oTracking.event', { detail: { category: 'element', action: 'click' }, bubbles: true}));">Send an event</button>
-</body>
-</html>
-```
+You can configure o-tracking to listen for `oTracking.page` and `oTracking.event` events instead of using the `page` and `event` methods.
 
-## Events
+##### oTracking.page
 
-`o-tracking` captures events automatically when initialised with the methods [outlined above](#useage). In addition, `o-tracking` will capture custom `oTracking.page` and `oTracking.event` events:
+To make o-tracking listen for `oTracking.page` events, you call `oTracking.page.init()`:
+```js
+// Tell o-tracking to listen for `oTracking.page` events
+oTracking.page.init();
 
-- `oTracking.page`
+// Now we can send page events and o-tracking will track the data within them.
+const pageData = { content: { uuid: 'abc-123', barrier: 'PREMIUM' };
+document.body.dispatchEvent(new CustomEvent('oTracking.page', { detail: pageData}, bubbles: true}));
+```
 
-    Send a page view event
+##### oTracking.event
 
-    ```js
-    document.body.dispatchEvent(new CustomEvent('oTracking.page', { detail: { content: { uuid: 'abc-123', barrier: 'PREMIUM' }}, bubbles: true}));
-    ```
-- `oTracking.event`
+To make o-tracking listen for `oTracking.event` events, you call `oTracking.event.init()`:
+```js
+// Tell o-tracking to listen for `oTracking.event` events
+oTracking.event.init();
 
-    Send a normal event
+// Now we can send custom events and o-tracking will track the data within them.
+const customData = { category: 'video', action: 'play', id: '512346789', pos: '10' };
+document.body.dispatchEvent(new CustomEvent('oTracking.event', { detail: customData}, bubbles: true}));
+```
 
-    ```js
-    document.body.dispatchEvent(new CustomEvent('oTracking.event', { detail: { category: 'video', action: 'play', id: '512346789', pos: '10' }, bubbles: true}));
-    ```
+### Example implementations
 
-## Parameters
+- [ft.com](docs/ftcom_example.md)
+- [membership](docs/membership_example.md)
```

Comment:
The line after this "Events are decorated with config values [...]" looks like its part of the "Example implementations" section. Needs a new heading?

Reply:
I'll place that content in the event.md file 👍